modules/apps/frontend-js/frontend-js-svg4everybody-web/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"dependencies": {
+		"svg4everybody": "^2.1.9"
```

Comment:
I doubt they'll be releasing new versions of `svg4everybody` but maybe pin this down?